holder.yml

```diff
@@ -10,10 +10,106 @@ info:
     name: GitHub Source Code
     url: https://github.com/w3c-ccg/vc-api
 paths:
+  /credentials/{id}:
+    get:
+      tags:
+       - Credentials
+      summary: Gets a credential or verifiable credential by ID
+      operationId: getCredential
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            format: uuid
```

Comment:
```suggestion
```

Reply:
Lets make sure to allow xss / command injection in our identifiers.

Reply:
+1 manu - added an `ObjectId` that matches that transaction id